front/app/components/Form/Components/ButtonBar.tsx

```diff
@@ -25,7 +25,14 @@ export default memo(
     onSubmit,
     apiErrors,
   }: Props) => (
-    <Box width="100%" background="#fff" border-top="solid 1px #ddd">
+    <Box
```

Comment:
Is it ok to change the `<Box>` in this file, which is probably used in different parts of the application?

Reply:
Yes, this is just used for the idea form (and now survey rendering as they are using the same components). The bug has been present for awhile, so this fixes the issue.

Reply:
Great.